dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -1,4 +1,6 @@
 - @page_title = t('activerecord.attributes.user.edit_header')
+- no_email = current_user.student?
+- no_email &= (current_user.parent_managed_account? || current_user.under_13? || params[:noEmail] == 'true')
```

Comment:
Can this be a property on `User`, ideally with tests? It seems awfully complex to be constructing in the view.